frontend/src/app/globals.css

```
@@ -50,3 +50,24 @@ canvas {
 ::-webkit-scrollbar-thumb:hover {
   background: #7d6b5a;
 }
+
+@keyframes bookshelfZoom {
```

Stylelint (17.12.0)

[error] Expected keyframe name "bookshelfZoom" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)